docs/reference/tctl/activity/complete.md

```diff
@@ -7,21 +7,15 @@ tags:
   - tctl
 ---
 
-<!-- prettier-ignore -->
-import * as WhatIsAnActivityExecution from '/docs/content/what-is-an-activity-execution.md'
-import * as WhatIsAWorkflowId from '/docs/content/what-is-a-workflow-id.md'
-import * as WhatIsARunId from '/docs/content/what-is-a-run-id.md'
-import * as WhatIsAnActivityId from '/docs/content/what-is-an-activity-id.md'
+The `tctl activity complete` command completes an [Activity Execution](/docs/content/what-is-an-activity-execution).
 
-The `tctl activity complete` command completes an <preview page={WhatIsAnActivityExecution}>Activity Execution</preview>.
+`tctl activity complete [<modifiers>]`
 
-`tctl activity complete <options> <arguments...>`
-
-The following options modify the behavior of the command.
+The following modifiers control the behavior of the command.
 
 ### `--workflow_id`
 
-How to specify the <preview page={WhatIsAWorkflowId}>Workflow Id</preview> of an <preview page={WhatIsAnActivityExecution}>Activity Execution</preview> to complete using tctl.
+How to specify the [Workflow Id](/docs/content/what-is-a-workflow-id) of an [Activity Execution](/docs/content/what-is-an-activity-execution) to complete using tctl.
```

Comment:
With this specific command, I think the "How to" is at the command level - i.e. "How to provide the result and complete an Activity Execution using tctl"

It may not be the case for all - as we can map some how tos to a modifier level - but In this case seems a bit off.

Reply:
Good point; done.